plugins/ltac2/tac2ffi.mli

```diff
@@ -130,6 +130,10 @@ val of_sort : ESorts.t -> valexpr
 val to_sort : valexpr -> ESorts.t
 val sort : ESorts.t repr
 
+val of_rewstrategy : (Ltac_pretype.closed_glob_constr, Redexpr.red_expr Tactypes.delayed_open, Id.t) Rewrite.strategy_ast -> valexpr
+val to_rewstrategy : valexpr -> (Ltac_pretype.closed_glob_constr, Redexpr.red_expr Tactypes.delayed_open, Id.t) Rewrite.strategy_ast
+val rewstrategy : (Ltac_pretype.closed_glob_constr, Redexpr.red_expr Tactypes.delayed_open, Id.t) Rewrite.strategy_ast repr
```

Comment:
I had to rollback to structured AST because the monad was really sick of fixpoints.

I instantiate `closed_glob_constr` as `'constr` because this is what I receive when deserializing a `preterm`. Ideally I would use `delayed_open_constr`, but `strategy_of_ast` wants both glob and econstr and I can't retrieve the former from the latter. I guess the glob is needed for `StratFold`? Otherwise it seems redundant, unless I missed something.

Anyway, is this ok?